src/DiscordBot.js

```diff
@@ -234,8 +234,14 @@ class DiscordBot {
     // If this is the verify channel, we want to delete the message and just verify the user if they aren't an admin.
     if (
       server.getSetting("verifyChannel") === message.channel.id &&
-      message.cleanContent.toLowerCase() !==
-        message.guild.commandPrefix + "verify" &&
+      (
+        message.cleanContent.toLowerCase() !==
+          message.guild.commandPrefix + "verify" &&
+        message.cleanContent.toLowerCase() !==
+          message.guild.commandPrefix + "update" &&
+        message.cleanContent.toLowerCase() !==
+          message.guild.commandPrefix + "getroles"
+      ) &&
```

Comment:
Maybe a bit nit-picky, but this feels fairly verbose - especially if there will be anymore commands down the line!
Have you considered using an array & .includes?

Reply:
done, completely agree with you